test/nodes/test_generator.py

```diff
@@ -197,3 +197,22 @@ def test_build_prompt_within_max_length():
 
         assert len(prompt_docs) == 1
         assert prompt_docs[0] == documents[0]
+
+
+@pytest.mark.integration
```

Comment:
Should this be unit at this point?

Reply:
That means the mocking was wrong, when this happens you should fix it instead of just marking it integration. I have fixed it for you, please have a look at my commit.